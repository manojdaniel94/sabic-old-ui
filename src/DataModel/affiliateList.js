export const affiliateList = [
  {
    "affiliateId": 2,
    "affiliateName": "SAFCO"
  },
  {
    "affiliateId": 3,
    "affiliateName": "MTVERNON"
  },
  {
    "affiliateId": 4,
    "affiliateName": "YANPET"
  },
  {
    "affiliateId": 5,
    "affiliateName": "PETROKEMYA"
  },
  {
    "affiliateId": 7,
    "affiliateName": "KEMYA"
  },
  {
    "affiliateId": 8,
    "affiliateName": "SHARQ"
  },
  {
    "affiliateId": 9,
    "affiliateName": "YANSAB"
  },
  {
    "affiliateId": 10,
    "affiliateName": "UNITED"
  },
  {
    "affiliateId": 11,
    "affiliateName": "GELEEN"
  },
  {
    "affiliateId": 16,
    "affiliateName": "ibn zahr"
  },
  {
    "affiliateId": 19,
    "affiliateName": "SAUDI KAYAN"
  },
  {
    "affiliateId": 20,
    "affiliateName": "ARRAZI"
  },
  {
    "affiliateId": 21,
    "affiliateName": "IBN ZAHR"
  },
  {
    "affiliateId": 22,
    "affiliateName": "IBNSINA"
  },
  {
    "affiliateId": 28,
    "affiliateName": "sharqtemp"
  },
  {
    "affiliateId": 29,
    "affiliateName": "SABIC Agri-Nutrients"
  },
  {
    "affiliateId": 41,
    "affiliateName": "YANSAB_Copy"
  },
  {
    "affiliateId": 47,
    "affiliateName": "UNITED_Copy"
  },
  {
    "affiliateId": 48,
    "affiliateName": "ARRAZI_Copy"
  }
]