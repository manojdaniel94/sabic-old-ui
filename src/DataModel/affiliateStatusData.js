export const affiliateStatusData = [
    {
        "title": "Affiliate_cnt",
        "value": "19"
    },
    {
        "title": "Asset_cnt",
        "value": "464"
    },
    {
        "title": "HealthIndex",
        "value": "23 %"
    },
    {
        "title": "PM_compliance",
        "value": "55 %"
    },
    {
        "title": "ActiveAlert_cnt",
        "value": "208"
    },
    {
        "title": "UnderInvtAlert_cnt",
        "value": "1"
    },
    {
        "title": "OverdueAlert_cnt",
        "value": "207"
    }
]