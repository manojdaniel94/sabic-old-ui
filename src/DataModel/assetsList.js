export const assetList = [
  {
    "assetId": "41 K101",
    "assetName": "Natural Gas Compressor"
  },
  {
    "assetId": "41 K101",
    "assetName": "Natural Gas Compressor"
  },
  {
    "assetId": "41 K701",
    "assetName": "Refrigeration Gas Compressor"
  },
  {
    "assetId": "41 K601",
    "assetName": "Syn Gas Compressor"
  },
  {
    "assetId": "41 P301A",
    "assetName": "Semi Lean Solution Pump  A"
  },
  {
    "assetId": "41 P301B",
    "assetName": "Semi Lean Solution Pump  B"
  },
  {
    "assetId": "41 P301C",
    "assetName": "Semi Lean Solution Pump  C"
  },
  {
    "assetId": "41 P302A",
    "assetName": "Lean Solution Pump A"
  },
  {
    "assetId": "41 P302B",
    "assetName": "Lean Solution Pump B"
  },
  {
    "assetId": "41-R-103",
    "assetName": "Secondary Reformer"
  },
  {
    "assetId": "42-E-201",
    "assetName": "High Pressure Stripper"
  },
  {
    "assetId": "42-E-205",
    "assetName": "High Pressure Pool Condensor"
  },
  {
    "assetId": "42-E-302",
    "assetName": "Rectifying Column Recirculation Heater"
  },
  {
    "assetId": "42 K102",
    "assetName": "CO2 Compressor Turbine"
  },
  {
    "assetId": "42 P102A",
    "assetName": "Ammonia Pump A"
  },
  {
    "assetId": "42 P102B",
    "assetName": "Ammonia Pump B"
  },
  {
    "assetId": "42 P301A",
    "assetName": "HP Carbomate Pump A"
  },
  {
    "assetId": "42 P301B",
    "assetName": "HP Carbomate Pump B"
  },
  {
    "assetId": "42-R-201",
    "assetName": "High Pressure Reactor"
  }
]