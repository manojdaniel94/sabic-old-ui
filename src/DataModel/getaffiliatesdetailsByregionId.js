export const getaffiliatesdetailsByregionId = [
  {
    "regionid": 1,
    "affiliateId": 2,
    "affiliateName": "SAFCO",
	  "latitude" : "55.237049",
	  "longitude" : "68.55082767"
  },
  {
    "regionid": 1,
    "affiliateId": 5,
    "affiliateName": "PETROKEMYA",
	  "latitude" : "42.237049",
	  "longitude" : "60.55082767"
  },
  {
    "regionid": 1,
    "affiliateId": 4,
    "affiliateName": "YANPET",
	  "latitude" : "40.237049",
	  "longitude" : "40.55082767"
  },
   {
    "regionid": 2,
    "affiliateId": 7,
    "affiliateName": "KEMYA",
	  "latitude" : "27.04",
	  "longitude" : "60.55"
  },
   {
    "regionid": 2,
    "affiliateId": 8,
    "affiliateName": "SHARQ",
	  "latitude" : "30.04",
	  "longitude" : "49.55"
  },
  {
    "regionid": 3,
    "affiliateId": 9,
    "affiliateName": "YANSAB",
	  "latitude" : "27.04",
	  "longitude" : "1.55"
  },
  {
    "regionid": 3,
    "affiliateId": 10,
    "affiliateName": "Test1",
	  "latitude" : "20.07291",
	  "longitude" : "30.5764"
  },
  {
    "regionid": 3,
    "affiliateId": 11,
    "affiliateName": "Test2",
	  "latitude" : "30.60",
	  "longitude" : "20.03"
  },
  {
    "regionid": 4,
    "affiliateId": 12,
    "affiliateName": "Test3",
	  "latitude" : "37.0902",
	  "longitude" : "95.7129"
  },
   {
    "regionid": 4,
    "affiliateId": 13,
    "affiliateName": "Test4",
	  "latitude" : "39.000000",
	  "longitude" : "85.7129"
  },
   {
    "regionid": 4,
    "affiliateId": 14,
    "affiliateName": "Test5",
	  "latitude" : "32.000000",
	  "longitude" : "80"
  },
  {
    "regionid": 1,
    "affiliateId": 15,
    "affiliateName": "Test6",
	  "latitude" : "51.237049",
	  "longitude" : "45.55082767"
  },
]