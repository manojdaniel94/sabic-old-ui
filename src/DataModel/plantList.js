export const plantList = [
  {
    "plantId": 1,
    "plantName": "AN4"
  },
  {
    "plantId": 19,
    "plantName": "AN5"
  },
  {
    "plantId": 48,
    "plantName": "AN3-AMMONIA"
  },
  {
    "plantId": 49,
    "plantName": "SAN-IBB"
  }
]