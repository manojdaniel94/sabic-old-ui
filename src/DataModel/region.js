export const region = [
  {
    "regionid": 1,
    "regionName": "Middle East Africa",
    "affiliatesCount": 4,
    "latitude": 52.237049,
    "longitude": 49.55082767,
    "color":"#009fdf"
  },
  {
    "regionid": 2,
    "regionName": "Middle East Asia",
    "affiliatesCount":2,
    "latitude": 29.298528,
    "longitude": 42.55096,
    "color":"#ffa300"
  },
  {
    "regionid": 3,
    "regionName": "Algeria",
    "affiliatesCount": 3,
    "latitude": 28.0339,
    "longitude": 1.6596,
    "color":"#009fdf"
  },
  {
    "regionid": 4,
    "regionName": "United States",
    "affiliatesCount": 3,
    "latitude": 37.0902,
    "longitude": 95.7129,
    "color":"#ffa300"
  }
]