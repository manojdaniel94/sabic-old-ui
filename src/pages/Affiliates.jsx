import React, { useRef } from "react";
import AffiliatesContainer from "../Components/AffiliatesPage/AffiliatesContainer";
import { useLocation } from 'react-router-dom';

const Affiliates = () => {

    const location = useLocation();
    // console.log("location.state.name", location.state.data);
    return (
        <>
            {/* <AffiliatesContainer affiliatesData={location.state.data} /> */}
            <AffiliatesContainer />
        </>

    );
};

export default Affiliates;