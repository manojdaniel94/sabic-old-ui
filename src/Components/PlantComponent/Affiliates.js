import Container from "../Container";
import Affiliate from "./Affiliate";
import { data } from "../../DataModel/data";
import { useCommonDispatch } from "../../context/common/context";
import { useNavigate } from "react-router-dom";

const Affiliates = () => {
  const dispatchCommon = useCommonDispatch();

  const _getCommonAction = (item) =>
    dispatchCommon({ type: "ADD_BREAD_CRUMBS", payload: item });
  let navigate = useNavigate();

  const onClickFunction = (key) => {
    navigate(`/${key}`);
  };

  const handleOnClick = (item) => {
    console.log(item);
    onClickFunction("assets");
    _getCommonAction({ title: "OLF1", key: "assets" });
  };

  return (
    <div className="affiliates-outer">
      <Container variant="affiliates">
        {data.map((data) => (
          <Affiliate
            name={data.name}
            image={data.image}
            handleOnClick={handleOnClick}
          />
        ))}
      </Container>
    </div>
  );
};

export default Affiliates;
