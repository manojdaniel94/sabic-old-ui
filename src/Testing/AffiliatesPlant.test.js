import React from "react";
import { render, cleanup } from "@testing-library/react";
import Affiliates from "../Components/PlantComponent/Affiliates";
import { data } from "../DataModel/data";

afterEach(cleanup);

describe("Affiliates component", () => {
  it("renders without crashing", () => {
    const { asFragment } = render(<Affiliates />);
    expect(asFragment()).toMatchSnapshot();
  });

  {
    /*} it("renders the data correctly", () => {
    const { getByText, getByAltText } = render(<Affiliates />);
    const firstData = data[0];
    const name = getByText(firstData.name);
    const image = getByAltText(firstData.name);

    expect(name).toBeInTheDocument();
    expect(image).toHaveAttribute("src", firstData.image);
  });*/
  }
});
