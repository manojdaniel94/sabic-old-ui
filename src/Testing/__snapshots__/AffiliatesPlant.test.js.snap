// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Affiliates component renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="affiliates-outer"
  >
    <div
      id="affiliates"
    >
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
      <div
        class="affiliates-box"
      >
        <div>
          OLF 1
        </div>
        <div>
          <span>
            HEALTH INDEX
          </span>
          <span
            class="green"
          >
            29%
          </span>
        </div>
        <div>
          <span>
            PM COMPLIANCE
          </span>
          <span
            class="blue"
          >
            70%
          </span>
        </div>
        <div>
          <span>
            ACTIVE ALERTS
          </span>
          <span
            class="red"
          >
            87
          </span>
        </div>
        <div>
          <span>
            UNDER INVESTIGATION
          </span>
          <span
            class="blue"
          >
            1
          </span>
        </div>
        <div>
          <span>
            OVERDUE INVESTIGATION
          </span>
          <span
            class="red"
          >
            86
          </span>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
